### ISSUE TITLE
Can i run it in linux arm64?

### ISSUE THREAD

**Issue by memidhun** [Genymobile/scrcpy#1976]
I want to run this in my kali linux arm64 ..How can i do that?Its not available from the sudo apt install command ..gives error scrcpy not found in snap store same isse not for arm64 cpu ..How can i do it

**Comment by rom1v** [comment]
Yes, follow the build instructions: https://github.com/Genymobile/scrcpy/blob/master/BUILD.md

**Comment by memidhun** [comment]
> Yes, follow the build instructions: https://github.com/Genymobile/scrcpy/blob/master/BUILD.md

oh thankyou so much i am kind of noob in linux ..So will it be hard for me?

**Comment by memidhun** [comment]
![error](https://user-images.githubusercontent.com/67965127/102203987-2ed12200-3ec1-11eb-9564-46d65eb977e4.png)
I am getting this following error when i run  ninja -Cx

**Comment by rom1v** [comment]
To keep things simple, use the [prebuilt server](https://github.com/Genymobile/scrcpy/blob/master/BUILD.md#prebuilt-server), so you just have to build the client.

_Also, please post console content as text instead of screenshot._

**Comment by memidhun** [comment]
> To keep things simple, use the [prebuilt server](https://github.com/Genymobile/scrcpy/blob/master/BUILD.md#prebuilt-server), so you just have to build the client.
> 
> _Also, please post console content as text instead of screenshot._

got it but now its giving me an other error kali@kali:~/Desktop/scrcpy/x/app$ scrcpy
bash: scrcpy: command not found
kali@kali:~/Desktop/scrcpy/x/app$ ./scrcpy
INFO: scrcpy 1.16 <https://github.com/Genymobile/scrcpy>
stat: No such file or directory
ERROR: '/usr/local/share/scrcpy/scrcpy-server' does not exist or is not a regular file

**Comment by memidhun** [comment]
Oh its working....Excellent! Thankyou so much
![scrcpy](https://user-images.githubusercontent.com/67965127/102216305-d73bb200-3ed2-11eb-9103-9d21256257d4.png)

**Comment by KateLevien** [comment]
I got the same problem in Debian. How did u resolve that? Thank you for letting me know in advance.

**Comment by TheRafaelFarias** [comment]
Install with snap and will work using the latest version

`snap install scrcpy`

In case you don't have snap is just use the guide on the snap website, and then run the command above

https://snapcraft.io/docs/installing-snap-on-debian

**Comment by mehran-mousavi** [comment]
Unofficial pre-built binaries of [scrcpy](https://github.com/Genymobile/scrcpy) for 64-bit ARM Linux (aarch64 / arm64). The upstream project does not ship ready-made ARM64 Linux packages; this repository publishes builds so you can install and run scrcpy on Raspberry Pi boards, ARM Chromebooks, Asahi Linux, and other AArch64 distributions without compiling from source.

    Current build: [scrcpy 3.3.4](https://github.com/Genymobile/scrcpy/releases/tag/v3.3.4)
    Target: Linux aarch64
    Built on: Debian 13 (Trixie)

These artifacts are not affiliated with Genymobile or the official scrcpy maintainers. For source code, documentation, and issue reporting about scrcpy itself, use the [official repository](https://github.com/Genymobile/scrcpy).


## Download

**Always-current archive (from the default branch):**

[https://github.com/mehran-mousavi/scrcpy-linux-arm64-aarch64-prebuild/releases/latest/download/scrcpy-aarch64-linux-aarch64.tar.gz](https://github.com/mehran-mousavi/scrcpy-linux-arm64-aarch64-prebuild/releases/latest/download/scrcpy-aarch64-linux-aarch64.tar.gz)

Each push to `main` (or `master`) refreshes the rolling **continuous** release and replaces that file. Checksums are attached on the same release page as `SHA256SUMS.txt`.

1. Open **[Releases](https://github.com/mehran-mousavi/scrcpy-linux-arm64-aarch64-prebuild/releases)** for notes, checksums, and history.
2. Or use the direct link above, then extract — contents unpack under `scrcpy-aarch64/`.
3. The tree matches **scrcpy 3.3.4** when your committed binaries match that version (see release notes on each build).
